### ISSUE TITLE
Info regarding number of keypoints for training

### ISSUE THREAD

**Issue by vishalsharbidar** [magicleap/SuperGluePretrainedNetwork#124]
Hello,
Thanks for the work and great paper.

While training, I noticed that the number of keypoints for img1 and img2 is constant. The loss function is also built on this assumption.
Can you please let me know what may happen if I use a different number of keypoints for img1 and img2?
Would it have an adverse effect on the training?
Should I use batch size =1 in this scenario? 

Thank you.

**Comment by sarlinpe** [comment]
Section 4 of the paper:
> To allow for data augmentation, SuperPoint detect and describe steps are performed on-the-fly as batches during training. A number of random keypoints are further added for efficient batching and increased robustness.

We train with an identical number of keypoints in each image but the model handles well unbalanced keypoints at test time.
